### PR TITLE
feat: add list command to edge_application

### PR DESCRIPTION
**Pull request by MaxwelMazur**
- create command list to edge application
- list all async
- unit test

### CODE REVIEW

File: messages/edge_applications/messages.go [hunk 1]
```diff
@@ -66,6 +66,11 @@ var (
 	EdgeApplicationsAWSAcess   = "AWS_ACCESS_KEY_ID: "
 
 	//CRUD
+	//list cmd
+	EdgeApplicationsListUsage            = "list [flags]"
+	EdgeApplicationsListShortDescription = "Displays your account's Edge Applicationss"
+	EdgeApplicationsListLongDescription  = "Displays all Applicationss in the user account’s Edge Applicationss library"
```
Comment (PatrickMenoti) [inline messages/edge_applications/messages.go:72]: Applications

File: pkg/cmd/edge_applications/list/list.go [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	opts := &contracts.ListOptions{}
+	cmd := &cobra.Command{
+		Use:           msg.EdgeApplicationsListUsage,
+		Short:         msg.EdgeApplicationsListShortDescription,
+		Long:          msg.EdgeApplicationsListLongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true, Example: heredoc.Doc(`
+		$ azioncli edge_functions list --details
```
Comment (PatrickMenoti) [inline pkg/cmd/edge_applications/list/list.go:27]: edge_applications

**Review by PatrickMenoti — CHANGES_REQUESTED** [review]

File: pkg/cmd/edge_applications/list/list.go [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	opts := &contracts.ListOptions{}
+	cmd := &cobra.Command{
+		Use:           msg.EdgeApplicationsListUsage,
+		Short:         msg.EdgeApplicationsListShortDescription,
+		Long:          msg.EdgeApplicationsListLongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true, Example: heredoc.Doc(`
+		$ azioncli edge_applications list --details
+		$ azioncli edge_applications ist --order_by "id"
```
Comment (PatrickMenoti) [inline pkg/cmd/edge_applications/list/list.go:28]: list

File: pkg/cmd/edge_applications/list/list.go [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+			return nil
+		},
+	}
+
+	cmdutil.AddAzionApiFlags(cmd, opts)
+	cmd.Flags().BoolP("help", "h", false, msg.EdgeApplicationsListHelpFlag)
+	return cmd
+}
+
+func insertTheFlagDetails(cmd *cobra.Command) bool {
```
Comment (PatrickMenoti) [inline pkg/cmd/edge_applications/list/list.go:60]: if you really want to create a function for checking if the flag is present, you can do it generically, like you did on init command. 

`func hasThisFlag(c *cobra.Command, flag string) bool {
	return c.Flags().Changed(flag)
}`

File: pkg/cmd/edge_applications/list/list_test.go [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package list
+
+import (
+	"github.com/aziontech/azion-cli/pkg/httpmock"
+	"github.com/aziontech/azion-cli/pkg/testutils"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestList(t *testing.T) {
```
Comment (PatrickMenoti) [inline pkg/cmd/edge_applications/list/list_test.go:10]: maybe another test? an error or no return

Comment (MaxwelMazur) [reply pkg/cmd/edge_applications/list/list_test.go:10]: Só testei o fluxo de retorno sem retorno de erro

File: pkg/cmd/edge_applications/list/list.go (hunk 4 above)
Comment (MaxwelMazur) [reply pkg/cmd/edge_applications/list/list.go:60]: A ideia é só deixar mais legivel, acredito que fique bom e legivel

**Review by PatrickMenoti — APPROVED** [review]
lgtm

**Pull request merged**